python/web/index.py

```diff
@@ -0,0 +1,139 @@
+import os
+import flask
```

Comment:
Is flask even necessary here?  I know we're using many 3rd party libraries to demonstrate how to connect to services, but that's out of necessity.  Eg, we just use bare PDO for PHP SQL connections.  Could this be done without an extra framework dependency?